roles/openstack/neutron/tasks/main.yml

```diff
@@ -0,0 +1 @@
+- include: "{{ action }}.yml"
```

Comment:
Adding some comment on top of this file explaining what this task is about is more then welcome.